test/suites/storage/reboot.robot

```diff
@@ -28,10 +28,12 @@ Rebooting Healthy System Should Keep Functional PVC
 *** Keywords ***
 Test Case Setup
     [Documentation]    Prepare the cluster env and test pod workload.
+    ${ns}=    Create Unique Namespace
+    Set Test Variable    \${NAMESPACE}    ${ns}
     Oc Create    -f ${SOURCE_POD} -n ${NAMESPACE}
     Named Pod Should Be Ready    ${POD_NAME_STATIC}
 
 Test Case Teardown
     [Documentation]    Clean up test suite resources
     Oc Delete    -f ${SOURCE_POD} -n ${NAMESPACE}
-    Named Pod Should Be Deleted    ${POD_NAME_STATIC}    timeout=120s
+    Oc Delete    ns ${NAMESPACE}
```

Comment:
Maybe use this one? https://github.com/openshift/microshift/blob/main/test/resources/kubeconfig.resource#L82